test/extended/util/jenkins/monitor.go

```diff
@@ -61,6 +61,11 @@ func (jmon *JobMon) Await(timeout time.Duration) error {
 		}
 
 		ginkgolog("Jenkins job %q build complete:\n%s\n\n", jmon.jobName, body)
+		// If Jenkins job has completed, output its log
+		body, status, err = jmon.j.GetResource("job/%s/%s/consoleText", jmon.jobName, jmon.buildNumber)
+		if err == nil && status == 200 {
+			ginkgolog("Jenkins job %q log:\n%s\n\n", jmon.jobName, body)
+		}
```

Comment:
if you fail to get the logs for some reason, it's probably worth logging that fact too (including what the error/error code was)